app/about_us/OurCrew.js

```diff
@@ -0,0 +1,32 @@
+import styles from "../../components/cards/OurCrewCard.module.css";
+
+import OurCrewData from "@/data/ourCrewData";
+import OurCrewCard from "@/components/cards/OurCrewCard";
+
+const OurCrew = () => {
+    return (
+        <>
+            <h1 className={styles.ourCrewCardsTitle}>Our Crew</h1>
```

Comment:
There should be only `h1` on each page. Heading levels should always follow a logical order of :
h1 > h2 > h3 ....